sdl/v1.go

```diff
@@ -226,11 +200,11 @@ func (sdl *v1) Manifest() (manifest.Manifest, error) {
 				Image: svc.Image,
 				Args:  svc.Args,
 				Env:   svc.Env,
-				Unit: types.Unit{
-					CPU:     uint32(compute.CPU),
-					Memory:  uint64(compute.Memory),
-					Storage: uint64(compute.Storage),
-				},
+				// Unit: types.Unit{
+				// 	CPU:     uint32(compute.CPU),
+				// 	Memory:  uint64(compute.Memory),
+				// 	Storage: uint64(compute.Storage),
+				// },
```

Comment:
what to do with all of the commented out sections in here?

fwiw, we don't need to maintain backwards compatibility